models.js

```diff
@@ -80,4 +80,14 @@ function selectCommentsByArticleId(articleId) {
                 return [];});
         }
         
-        module.exports = { selectTopics, selectArticlesById, selectAllArticles, selectCommentsByArticleId };
\ No newline at end of file
+function createComments(author, body, article_id, votes){
```

Comment:
Clients should not be able to insert votes, votes are defaulted to 0 which makes sense, otherwise a client could add an article with 100000 votes and ruin the voting system

Reply:
Thank you for pointing those out.